src/main/resources/avro/bdg.avdl

```diff
@@ -58,6 +58,11 @@ record Contig {
 }
 
 record AlignmentRecord {
+ 
+  /**
+   Read number within the array of fragment reads.
+   */
+  union { int, null } readNum = 0;
```

Comment:
As written, it is correct, no? The first type needs to be the type of the default value --> e.g., `0` is an `int`.